composer.json

```diff
@@ -36,9 +36,6 @@
     "autoload": {
         "psr-0": {"IsoCodes": "src/"}
     },
-    "config": {
-        "bin-dir": "bin"
```

Comment:
I updated the bin path on script too.

I had to do this. See: https://github.com/ronanguilloux/IsoCodes/pull/57#issuecomment-114264368

BTW, I have another suggestion to do about this on another PR.